lib/index.js

```diff
@@ -175,16 +175,14 @@ const getCombinablePRs = async function* (
         apiRef
       );
 
-      const isNotAllGreenOrSkipped = (check) => {
+      const isAllGreenOrSkipped = (check) => {
         const { conclusion } = check;
-        const isNotSuccessfull = conclusion !== "success";
-        const isNotSkipped = conclusion !== "skipped";
-
-        return !allowSkipped ? isNotSuccessfull : isNotSuccessfull && isNotSkipped;
+        const isSuccessfull = conclusion === "success";
+        const isSkipped = conclusion === "skipped";
+        return !allowSkipped ? isSuccessfull : isSuccessfull || isSkipped;
       }
 
-
-      if (checks.some(isNotAllGreenOrSkipped)) {
+      if (checks.some(c => !isAllGreenOrSkipped(c))) {
```

Comment:
Somewhat nitty, but if we're going to pass in an arrow function here, we should lift `isAllGreenOrSkipped` out of this function body and pass in both `c` & `allowSkipped`.